packages/model-viewer/src/features/controls.ts

```diff
@@ -361,6 +366,18 @@ export const ControlsMixin = <T extends Constructor<ModelViewerElementBase>>(
     @property({type: Boolean, attribute: 'enable-pan'})
     enablePan: boolean = false;
 
+    @property({type: Number, attribute: 'pan-key-increment'})
+    panKeyIncrement: number = DEFAULT_PANKEYINCREMENT;
+
+    @property({type: Number, attribute: 'pan-key-multiplier'})
+    panKeyMultiplier: number = DEFAULT_PANKEYMULTIPLIER;
+
+    @property({type: Number, attribute: 'pan-mouse-multiplier'})
+    panMouseMultiplier: number = DEFAULT_PANMOUSEMULTIPLIER;
+
+    @property({type: Number, attribute: 'orbit-key-multipler'})
+    orbitKeyMultipler: number = DEFAULT_ORBITKEYMULTIPLIER;
```

Comment:
I am not excited about expanding our API surface this much for such a small feature. Let's talk through design first.

Reply:
Where do I "You need to leave a comment indicating the requested changes."???